package.json

```diff
@@ -52,7 +52,7 @@
     "eslint-watch": "^4.0.1",
     "flow-bin": "0.75.0",
     "glob": "^7.1.2",
-    "jest": "^23.3.0",
+    "jest": "^23.4.1",
```

Comment:
This isn't actually necessary.  I was running into issues with some of `vite`'s deps interfering with `jest`.  I think it was `frametalk` which had an old version of `jest` as a dependency.